torch/_inductor/ir.py

```diff
@@ -3630,21 +3630,42 @@ def __init__(
 
         self.op_overload = kernel
 
-        op_overload_packet = (
-            kernel._overloadpacket
-            if isinstance(kernel, torch._ops.OpOverload)
-            else kernel
-        )
+        # TODO: Need to revisit schema matching to find the correct OpOverload from OpOverloadPacket
```

Comment:
For the future, I don't think inductor should ever get an OpOverloadPacket and we should maintain that invariant. AOTAutograd should normalize things into OpOverload/HigherOrderOperator (and we've defined "ATen IR" to only consist of OpOverload/HigherOrderOperator), so is the only source of OpOverloadPacket additional passes after AOTAutograd and before we get here?

Reply:
It's because in _inductor/lowering.py, we do something like https://github.com/pytorch/pytorch/blob/7c1702f099f4301a3f124fc7bf697f3234d7f9d2/torch/_inductor/lowering.py#L1860 which passes `OpOverloadPacket` to `make_fallback`. We can turn that into `OpOverload` with `.default`, but that probably requires more cleaning in cpp codegen. I will separate into another PR.